tests/test_decoder.py

```diff
@@ -734,3 +736,132 @@ def test_parse_error():
 
     with pytest.raises(pvl.decoder.ParseError):
         pvl.load(io.BytesIO(b'foo'))
+
```

Comment:
Can you reproduce the following issue?

I've added the following to `test_decoder.py`

```
BAD_PDS_LABELS = glob.glob(os.path.join(BROKEN_DIR, "*.lbl"))

def test_load_all_bad_sample_labels():
    for filename in BAD_PDS_LABELS:
        label = pvl.load(filename, strict=False)
        assert isinstance(label, Label)
```

And it fails with the error:

```

    def test_load_all_bad_sample_labels():
        for filename in BAD_PDS_LABELS:
>           label = pvl.load(filename, strict=False)

tests/test_decoder.py:704:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pvl/__init__.py:89: in load
    return cls(**kwargs).decode(fp)
pvl/decoder.py:187: in decode
    module = PVLModule(self.parse_block(stream, self.has_end))
pvl/decoder.py:206: in parse_block
    statement = self.parse_statement(stream)
pvl/decoder.py:264: in parse_statement
    return self.broken_parameter_value(stream.lineno - 1)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <pvl.decoder.PVLDecoder object at 0x10b73aa90>, lineno = 2

    def broken_parameter_value(self, lineno):
        if self.strict:
            msg = "Broken Parameter-Value. Set strict to False to skip error"
>           raise ParseError(msg, None, lineno, None)
E           pvl.decoder.ParseError: Broken Parameter-Value. Set strict to False to skip error: line 2

pvl/decoder.py:143: ParseError
```

This is basically just comparing:

```
>>> import pvl
>>> a = pvl.load('tests/data/pds3/tiny4.lbl')
>>> b1 = pvl.load('tests/data/pds3/broken/broken1.lbl', strict=False)
```

I have that usage right yeah:  `strict=False`?